apps/backend/src/resolvers/mutations/AssignInstrumentsToTechnique.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Args,
+  ArgsType,
+  Ctx,
+  Field,
+  Mutation,
+  Resolver,
+  Int,
+} from 'type-graphql';
+
+import { ResolverContext } from '../../context';
+
+@ArgsType()
+export class AssignInstrumentsToTechniqueArgs {
+  @Field(() => [Int])
+  public instrumentIds: number[];
+
+  @Field(() => Int)
+  public techniqueId: number;
+}
+
+@Resolver()
+export class AssignScientsitsToInstrumentMutation {
```

Comment:
The Class name is conflicting with the purpose of this Mutation Resolver. Could you please re-check once?